CONTRIBUTING_DOCS.md

```diff
@@ -132,27 +132,84 @@ Examples:
 
 
-| URL                                           | Filepath                                         |
-|-----------------------------------------------|--------------------------------------------------|
-| `/getting-started/kubernetes/`                | `/getting-started/kubernetes/index.md`           |
-| `/getting-started/kubernetes/troubleshooting` | `/getting-started/kubernetes/troubleshooting.md` |
+| URL                                           | File path                                         |
+|-----------------------------------------------|---------------------------------------------------|
+| `/getting-started/kubernetes/`                | `/getting-started/kubernetes/index.md`            |
+| `/getting-started/kubernetes/troubleshooting` | `/getting-started/kubernetes/troubleshooting.md`  |
 
 
-## Linking content
+### Relative links and `page.version` variable
 
-All links should be absolute links. To link to versioned content, prefix all links with: `{{site.baseurl}}/{{page.version}}/`
+To link to other doc site content, use relative links prefixed with: `/{{page.version}}/`. This allows the content to port across documentation versions without link breakage. Example:
+
+```
+[Get started](/{{page.version}}/getting-started/)
+```
 
 > **Pro tip**: `page.version` will be inherited from the default set in `_config.yml` for the current page's directory.
 
+### Case sensitivity
+
+Do not include any uppercase letters in your links.
+
+### Anchor links
+
+An anchor link for each heading is automatically created. It consists of the title of the heading with each word separated by hyphens. Delete any slashes in the title. For example, to reference a heading titled "Working with the calico/kube-controllers container" on a page located at `https://docs.projectcalico.org/v3.0/reference/kube-controllers/configuration`, you would use the following:
+
+```
+/{{page.version}}/reference/kube-controllers/configuration#working-with-the-calicokube-controllers-container
+```
+
+## Canonical URLs
+
+Because the documentation site includes content for past versions as well as the latest version, it contains many duplicate pages. When Google indexes the site, it needs to know which copy we prefer. We use [jekyll-seo-tag](https://github.com/jekyll/jekyll-seo-tag) to add [canonical URLs](https://support.google.com/webmasters/answer/139066?hl=en) to each page. This helps us to ensure that the latest copy of the page comes up first when people search for information via Google.
+
+The jekyll-seo-tag gem adds a canonical URL containing the absolute path to the current page by default. If the canonical URL is _not_ the current page, the page should contain `canonical_url` metadata that contains the absolute path to the latest copy of the page. 
+
+You should _not_ need to modify the `canonical_url` metadata unless you are adding, deleting, or renaming a page.
+
+
```

Comment:
To summarize:
- Pages on the latest release don't need any special canonical metadata. 
- All other pages should include a canonical link to the latest version of that page (or final version if it was removed). This includes master pages.

This helped me understand. Perhaps you can work this info in here somehow?

Reply:
Actually, after we discussed, I realized that all pages should include the canonical_url...to allow the updates. So I have updated this page according to our latest and greatest.